integration/tests/cook/cli.py

```diff
@@ -139,6 +139,11 @@ class temp_config_file:
 
     def __init__(self, config):
         self.config = config
+        session_module = os.getenv('COOK_SESSION_MODULE')
+        if session_module:
```

Comment:
Should we use an `if-else` branch to initialize `self.config`

Reply:
Done